client/src/api/util.js

```diff
@@ -7,13 +7,24 @@ export const relog = () => {
 }
 
 export const request = (method, endpoint, data) => {
-  return fetch(config.apiEndpoint + endpoint, {
+  let body = null
+  let qs = ''
+  if (method === 'GET' && data) {
+    // encode data into the querystring
+    qs = Object.keys(data)
+      .filter(k => data[k] !== undefined)
+      .map(k => `${k}=${encodeURIComponent(data[k])}`)
+      .join('&')
+  } else {
+    body = data
+  }
+  return fetch(`${config.apiEndpoint}${endpoint}?${qs}`, {
     method,
     headers: {
       'Content-Type': 'application/json',
       Authorization: `Bearer ${localStorage.getItem('token')}`
     },
-    body: JSON.stringify(data)
+    body: body && JSON.stringify(body)
```

Comment:
We might want to do strict type checking against `null` incase some false-y values get passed in as data to a POST request?

Reply:
`body` should be an object though, since we're `JSON.stringify`ing it, and objects are truthy, so this should be fine provided the function is called correctly